_includes/promos/10tvpromo.html

```diff
@@ -0,0 +1,106 @@
+            line-height: 71px;">Thank you 10tv</h1>
+            <h4><br/>10tv has partnered with Can't Stop Columbus and has graciously<br />volunteered to give free advertizing space on their <a href="https://www.10tv.com/">website</a>.</h4>
+            <h4><br/>Offer valid until May 1st, 2020<br />To submit your business, fill out the form below.</h4>
+        </div>
+        <div class="col-md-3">  
+            <br><img style="max-width: 100%;
+            height: auto;" src="assets/img/UX/logos/10tv.jpg">
+        </div>
+    </div>
+    <br />
+    <br />
+    <br />
+    <div class="row">
+        <div class="col-md-6">
+            <p class="subtitle" style="display:inline">Submit Your Business</p>
+        </div>
+    </div>
+    <br />
+    <form name="promo" class="needs-validation"   novalidate accept-charset="UTF-8"> <!--accept-charset="UTF-8" action="https://formkeep.com/f/exampletoken" method="POST"-->
+        <div class = "row">
+            <div class="col-md-5">
+                <input type="text" required class="form-control promoField" name="name"  placeholder="Business Name">
+                <div class="invalid-feedback">
+                    Please type in the business name
+                  </div>
+                <input type="url" class="form-control promoField" name="website"  required placeholder="Website URL">
+                <div class="invalid-feedback">
+                    Please type in a valid website URL
+                  </div>
+            </div> 
+            <div class="col-md-5">
+                <input type="text" required name="promo_text"  class="form-control promoField"  id="promo_text"  maxlength="100" placeholder="Promotional Text">
+                <div class="invalid-feedback">
+                    Please type in a valid text
+                  </div>
+                    <span id="charLeft"  style="color:grey; padding-left:10px; font-size:14px;"><span id="rem_post" title="1000">0</span>/100 characters   </span>
+                
+            </div>
+        </div>
+        <div class = "row">
+            <br />
+        </div>
+        <div class = "row">
+            <div class="col-md-12 text-center align-center">
+                 <button class="btn btn-primary dark-purple"  style="width:25%;margin-bottom:10px;" type="submit" id="submitButton">Submit</button>
+            </div>
+        </div>
+      </form>
+    <div class = "row">
+        <div class="col-md-12 text-center">
+            <span id="submitText" class="submitText">Thank you! Your ad will be up within 24 hours.</span>
+        </div>
+    </div>
+    <br>
+  </div>
+  
+
+  <script>
+    var copyText = document.getElementById("description");
+    var charLeft = document.getElementById("rem_post");
+    var submitText = document.getElementById("submitText");
+    var submitButton = document.getElementById("submitButton");
+    var forms = document.getElementsByClassName('needs-validation');
+   // 
```

Comment:
Stray comment

Reply:
A wild comment appeared. Have to catch them all. Thanks for spotting this.